src/i_video.c

```diff
@@ -515,6 +510,17 @@ void I_StartTic (void)
         ev.data3 = y;
 
         D_PostEvent(&ev);
+        return;
+    }
+
+    if (window_focused)
+    {
+        I_ReadMouse();
```

Comment:
I noticed that before this PR mouse movement in the menu wouldn't carry over into the game but now it does for some reason.

Reply:
Hmm, I don't understand why. How did you check it? Do you mean the `ev_mouse` event?

Reply:
Example: enter menu, move mouse around, exit menu. Expected: view is stationary. Actual: view immediately shifts to a different angle.

I think it's because the next time `SDL_GetRelativeMouseState()` is called, it's a very large change compared to before the menu was activated.

Reply:
Maybe it needs to be called and the values immediately discarded upon returning to the game. Not sure about gamepad.

Reply:
> Edit: Call `SDL_GetRelativeMouseState(NULL, NULL);` somewhere.

Done in https://github.com/fabiangreffrath/woof/pull/1460/commits/fcc26e9cbbf733b96f5143ce8a8fbd7f48c831ab